File: PortfolioERP.Application/Features/Orders/OrderLineResponse.cs
namespace PortfolioERP.Application.Features.Orders;

public sealed record OrderLineResponse(
    int Id,
    int ProductId,
    string ProductCode,
    string ProductName,
    int Quantity,
    decimal UnitPrice,
    decimal DiscountPercentage,
    decimal DiscountAmount,
    decimal NetAmount,
    decimal VatPercentage,
    decimal VatAmount,
    decimal TotalAmount);

File: frontend/portfolioERP-web/src/app/features/orders/models/order-line-response.ts
export interface OrderLineResponse {
  id: number;
  productId: number;
  productCode: string;
  productName: string;
  quantity: number;
  unitPrice: number;
  discountPercentage: number;
  discountAmount: number;
  netAmount: number;
  vatPercentage: number;
  vatAmount: number;
  totalAmount: number;
}
